ansible/roles/devspace/tasks/devspace-install.yml

```diff
@@ -135,7 +137,7 @@
     line: "        auth_basic \"Restricted\";"
     insertafter: "    location / {"
     state: present
-  when: devspace_jenkins_nginx_username|length>0 and devspace_jenkins_nginx_password|length>0
+  when: devspace_jenkins_nginx_password and devspace_jenkins_nginx_username|length>0 and devspace_jenkins_nginx_password|length>0
```

Comment:
`devspace_jenkins_nginx_password` is defined in `defaults/main.yml` file, so you could simplfy this check by setting the default to `devspace_jenkins_nginx_password: ''` so you can just check the length here